heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -459,7 +460,11 @@ contains JSON:
       'X-API-Key': '12345-mykey-67890'
   check.response:
     status: 200
-    body: '{"status": "ok"}'
+    json:
+      - description: check status
```

Comment:
What is the purpose of the description? logging output?

As this is YAML I would more expect someone that wants to put details about it as a yaml comment.

Reply:
It's interpolated into the `error` field in the event when the check fails.

This is important because it means the user can associate a human readable error with the downtime message when they get, say, an SMS.

So, instead of getting "http check for elastic.co/foo failed" they can get one that says `JSON body did not match condition '%s' for monitor. Received JSON %+v`, where `%s` is the description.

When using multiple conditions this is even more important since it may not be apparent what caused the failure.

Reply:
Got it. So the description is sent with every event or just errors (need to check the code).

Reply:
@ruflin no, it's only sent on error. There are multiple conditions checked, and the first one that fails is the only one that creates an error.

Now that I think about it, that's probably a mistake. We should run all the conditions and list all of the condition failures.

I'll improve the PR by making the error message include a list of error messages. `JSON body did not match 2 conditions: "version greater than 1", "name must be valid". Received JSON {...}`